app/views/_masthead.html.erb

```diff
@@ -0,0 +1,37 @@
+<% if current_ability.current_user.staff? %>
+  <%= render '/shared/select_work_type_modal' if create_work_presenter.many? %>
+  <nav id="staff-navbar" class="navbar navbar-inverse navbar-static-top staff-navbar" role="navigation">
+    <div class="container-fluid">
+      <div class="collapse navbar-collapse" id="staff-navbar-collapse">
+        <%# TODO: make these collapse to hamburger on mobile? two hamburgers is weird but it takes up lots of space otherwise %>
```

Comment:
i don't know if this is your comment or sufia's, but I'm not worried about admin-only stuff on mobile, admin won't be working on mobile much/at all I think.